examples/dataflow-streaming-benchmark/README.md

```diff
@@ -0,0 +1,100 @@
+# Dataflow Streaming Benchmark
+
+When developing Dataflow pipelines, it's common to want to benchmark them at a specific QPS using
+fake or generated data. This pipeline takes in a QPS parameter, a path to a schema file, and 
+generates fake JSON messages matching the schema to a Pub/Sub topic at the rate of the QPS.
+
+## Pipeline
+
+[StreamingBenchmark](src/main/java/com/google/cloud/pso/pipeline/StreamingBenchmark.java) -
+A streaming pipeline which generates messages at a specified rate to a Pub/Sub topic. The messages 
+are generated according to a schema template which instructs the pipeline how to populate the 
+messages with fake data compliant to constraints.
+
+> Note the number of workers executing the pipeline must be large enough to support the supplied 
```

Comment:
This could likely trip up new users. Can we check the options in the main class to check that we have sufficient cores for the specified QPS?

Reply:
Looked into this. It doesn't appear the juice is worth the squeeze. Checking the options is failure prone because they are reliant on machineType which isn't validated until the job is submitted to the cloud. This adds a whole bunch of failure cases. For now, I would say let's merge this as folks are waiting on it and I can continue to look into if there's a cleaner way to accomplish what you're asking for.